src/ex5/dist/main.js

```diff
@@ -1,57 +1,56 @@
 class Main {
```

Comment:
you have some bugs, i noticed that some time I add new task and I won't see it until I'll refresh

Reply:
another issue is with the id's, they are not really uniq right now which cause problems in deletions etc..